### PR TITLE
Adding Django to Marketplace

### PR DESCRIPTION
**Pull request by n0vabyte**
Adding Django to Marketplace

### CODE REVIEW

File: apps/linode-marketplace-django/roles/post/templates/motd.j2 [hunk 1]
```diff
@@ -0,0 +1,7 @@
+*********************************************************
+Akamai Connected Cloud Django Marketplace App
+App URL: https://{{ _domain }}
+Credentials File: /root/.credentials
```
Comment (hmorris3293) [inline apps/linode-marketplace-django/roles/post/templates/motd.j2:4]: switch to /home/{{ username }}/.credentials

File: apps/linode-marketplace-django/roles/post/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+-
+# motd and creds gen
+- name: copy MOTD template to /etc/motd
+  template:
+    src: templates/motd.j2
+    dest: /etc/motd
+
+- name: writing jitsi creds into file
```
Comment (hmorris3293) [inline apps/linode-marketplace-django/roles/post/tasks/main.yml:8]: s/jitsi/django

File: apps/linode-marketplace-django/roles/post/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  copy:
+    dest: "{{ item.user }}/.credentials"
```
Comment (hmorris3293) [inline apps/linode-marketplace-django/roles/post/tasks/main.yml:10]: I like this idea, but we only document the /home/user/.credentials in the docs shortcode, so I'm not sure if we want to put sensitive creds in both locations without proper documentation. We just restructured this in all ansible guides so personally I would stick to just the home/user dir

File: apps/linode-marketplace-django/roles/post/templates/motd.j2 (hunk 1 above)
Comment (n0vabyte) [reply apps/linode-marketplace-django/roles/post/templates/motd.j2]: done

**Review by hmorris3293 — APPROVED** [review]
tested and deployed successfully :)

**Pull request merged**